app/server.js

```diff
@@ -0,0 +1,27 @@
+const http = require('http');
```

Comment:
@rodrigopavezi Can we use consistent quotes? They should be double. The linter should have caught this since it's specified as a rule.